eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver.php

```diff
@@ -265,14 +265,15 @@ private function warnAboutTooEarlyLoadedParams()
         }
 
         $logger = $this->container->get('logger');
-        foreach ($this->tooEarlyLoadedList as $param => $services) {
+        foreach ($this->tooEarlyLoadedList as $blame => $params) {
             $logger->warning(sprintf(
-                'ConfigResolver was used to load parameter "%s" before SiteAccess was loaded by services: %s. This can cause issues. '
-                . 'Try to use ConfigResolver lazily, '
+                'ConfigResolver was used by "%s" before SiteAccess was initialized, loading parameter(s) '
+                . '%s. As this can cause very hard to debug issues, please '
+                . 'try to use ConfigResolver lazily, '
                 . (PHP_SAPI === 'cli' ? 'make commands that rely on them lazy, ' : '')
```

Comment:
```suggestion
                . (PHP_SAPI === 'cli' ? 'make commands that rely on them being lazy, ' : '')
```

Reply:
That would change the sentence to indicate the command relies on the services being lazy
What I'm trying to say is `make the command, that rely directly/indirectly on config resolver parameters, lazy`, we can drop the fact that they directly or indirectly rely on config resolver parameters


So maybe we shorten it to `make the affected commands lazy` now that the output more clearly states which commands are causing this.

ok with you?

Reply:
Ah, I wasn't clear about the original meaning. Yes, `make the affected commands lazy` sounds much clearer to me.

Reply:
```suggestion
                . (PHP_SAPI === 'cli' ? 'make the affected commands lazy, ' : '')
```